ioctl/client.go

```diff
@@ -48,6 +48,8 @@ type (
 		// doing
 		GetAddress(in string) (string, error)
 		// doing
+		Address(in string) (string, error)
```

Comment:
What's the difference bewteen "GetAddress()" and "Address()" ?

Reply:
all comes from iotcl/util.go.`GetAddress():GetAddress get address from address or alias or context`,`Address():Address returns the address corresponding to alias. if 'in' is an IoTeX address, returns 'in'`.